packages/salesforcedx-vscode-core/package.json

```diff
@@ -150,7 +150,7 @@
         {
           "id": "metadata",
           "name": "Metadata",
-          "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking"
```

Comment:
Org Browser is the only consumer of this setting.  I would propose a follow up WI to remove this setting entirely, and update the consumer of the other like setting, 'sfdx:default_username_has_change_tracking' to use the newer `getWorkspaceOrgType()` function instead.